### ISSUE TITLE
Where are the PureScript communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/purescript#90]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is PureScript supported by one or more large organizations?
* Does PureScript have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about PureScript? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for PureScript? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in PureScript? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in PureScript?
* Are there popular newsletters for PureScript?
* Is PureScript taught at programming bootcamps? (If so, what are some examples?)
* Is PureScript taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about PureScript?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by spicydonuts** [comment]
Communities:
- [FP Chat Slack #purescript](https://fpchat-invite.herokuapp.com/)
- [PureScript Language Forum](https://purescript-users.ml/)

No official manager that I know of.

PureScript Conf happens as part of Lambda Conf: https://www.meetup.com/lambdaconf/events/250698565/
LA PureScript Meetup: https://www.meetup.com/LA-PureScript/

Past recorded meetups and talks: https://www.youtube.com/channel/UCPtHLGu_WXh-OvX8NAVtDEw

I don't know of any groups specifically focused on underrepresented groups, but the communities listed above should be welcoming to all. I also don't of of specific mentor groups, but many have offered via Twitter (including myself, link on my profile).

Newsletters:
- [PureScript News (mailing list)](http://purescript.news/)

I don't know of any bootcamps or universities teaching PureScript, though some universities teach Haskell and it's very similar.

**Comment by kytrinyx** [comment]
Thanks @spicydonuts, this is really helpful.

**Comment by kytrinyx** [comment]
I've got what I need for now; closing this out. Thank you! 🌼
